src/driver/gen_rpc_driver_ssl.erl

```diff
@@ -62,17 +100,17 @@ listen(Port) when is_integer(Port) ->
     SslOpts = merge_ssl_options(server, undefined),
     ssl:listen(Port, SslOpts).
 
--spec accept(ssl:sslsocket()) -> ok | {error, term()}.
+-spec accept(ssl:sslsocket()) -> {ok, ssl:sslsocket()} | {error, term()}.
 accept(Socket) when is_tuple(Socket) ->
     {ok, TSocket} = ssl:transport_accept(Socket, infinity),
-    case ssl:ssl_accept(TSocket) of
-        ok ->
-            {ok, TSocket};
+    case ssl:handshake(TSocket) of
+        {ok, SSocket} ->
+            {ok, SSocket};
```

Comment:
I'd replace the `ssl:transport_accept/2` + `ssl:handshake/1` calls with a single `ssl:accept/2` - it would definitely count as modernizing.

Also, `ssl:transport_accept/2` might return `{error, _}`s.

But I'll leave it up to you.

Reply:
Wait, there is no such thing (`ssl:accept/2`.)

Where did this idea come from?

Oh well. Nothing to see, here.

Reply:
(I can't resolve my own conversations, apparently)

Reply:
(Ah, actually I can, but only after I finish a review.)